src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.ts

```diff
@@ -26,7 +27,21 @@ export class ExamParticipationSummaryComponent {
 
     collapsedExerciseIds: number[] = [];
 
-    constructor() {}
+    courseId: number;
+
+    constructor(private route: ActivatedRoute) {}
+
+    /**
+     * Initialise the courseId from the current url
+     */
+    ngOnInit(): void {
+        // courseId is not part of the exam or the exercise
+        this.courseId = Number(this.route.snapshot.paramMap.get('courseId'));
+    }
+
+    get isPublished() {
```

Comment:
If you consider changing the naming as suggested in a previous comment you might also adjust it here.

Reply:
Done